src/components/SearchBar/hooks.ts

```diff
@@ -1,30 +1,69 @@
+import _ from 'lodash';
 import { useCallback, useMemo, useState } from 'react';
 
-import { ColumnFilterValue, DateColumnFilterValue, SearchBarData, SearchBarProps } from './types';
+import { Resource } from '@beda.software/aidbox-types';
+
+import { LoadResourceOption } from 'src/services/questionnaire';
+
+import {
+    ColumnFilterValue,
+    DateTypeColumnFilterValue,
+    isDateColumn,
+    isReferenceColumn,
+    isStringColumn,
+    isStringColumnFilterValue,
+    isDateColumnFilterValue,
+    isReferenceColumnFilterValue,
+    ReferenceTypeColumnFilterValue,
+    SearchBarData,
+    SearchBarProps,
+    StringTypeColumnFilterValue,
+} from './types';
 
 export function useSearchBar(props: SearchBarProps): SearchBarData {
     const { columns } = props;
 
     const defaultFiltersValues = useMemo(() => {
-        return columns.map((column) => ({ column }));
-    }, [columns]);
+        return columns.map((column) => {
```

Comment:
Try to read of `as` operator and type conversion

Reply:
done